uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/BaseHeadlessTest.php

```diff
@@ -8,10 +8,10 @@
 
   public function setUpHeadless() {
     return \Civi\Test::headless()
-      ->installMe(__DIR__)
       ->install('uk.co.compucorp.civicrm.hrcore')
       ->install('org.civicrm.hrjobcontract')
       ->install('uk.co.compucorp.civicrm.hrcomments')
+      ->installMe(__DIR__)
```

Comment:
It's all a matter of dependencies. The l&a extension (installed with `installMe()`) depends on the other three, so it makes sense to have them installed first. Currently, the order of installation would not cause any problems, but it can in the future. For example, part of the l&a installation process might call an API from any of the other extensions and this call will fail because that extension is not available yet.